src/foremast/awslambda/api_gateway_event/api_gateway_event.py

```diff
@@ -79,13 +79,7 @@ def find_resource_id(self):
             if resource['path'] == self.trigger_settings['resource']:
                 resource_id = resource['id']
                 self.log.info("Found Resource ID for: %s", resource['path'])
-                self.attach_method(resource_id)
-                break
-        else:
-            resource_id = self.create_resource(parent_id=parent_id)
-            self.attach_method(resource_id)
-
-        return resource_id
+        return resource_id, parent_id
```

Comment:
This is fine, but I would like to see a follow up Pull Request. We can define the class attributes in `__init__()`. Then this method should assign the class attributes directly.

I have no issues with returning the values. Sijis and I like when functions and methods return at least something (makes testing easier). I would suggest using a `collections.namedtuple` in the future.